index.bs

```diff
@@ -7914,10 +7904,28 @@ Constructors</h4>
 				[[mediacapture-streams#mediastream|MediaStream]] that has at least one
 				[[mediacapture-streams#mediastreamtrack|MediaStramTrack]] whose
 				<code>kind</code> attribute has the value <code>"audio"</code>,
-				throw an {{InvalidStateError}} and abort these steps.
-			3. Let <var>node</var> be a new {{MediaStreamAudioSourceNode}}
+				throw an {{InvalidStateError}} and abort these steps. Else, let
+				this stream be <var>inputStream</var>.
+			3. Let <var>tracks</var> be the list of all
+				[[mediacapture-streams#mediastreamtrack|MediaStreamTrack]]s of
+				<var>inputStream</var> that have a <code>kind</code> of
+				<code>"audio"</code>.
+			4. Sort the elements in <var>tracks</var> based on their <code>id</code>
+				attribute using lexicographic ordering on sequences of code unit
```

Comment:
code unit or code point?  They are different.  But  I see that Javascript sorts by code unit (which seems really odd), so we're good here.

Reply:
Yes I'm purposefully using this term here so that it's aligned with what JavaScript's `sort` is doing.